{
  try {
    foo();
  }
 catch (  Exception e) {
    e.printStackTrace();
  }
}
